src/Raven.Client/ServerWide/Operations/Configuration/ServerWideBackupConfiguration.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Raven.Client.Documents.Operations.Backups;
+
+namespace Raven.Client.ServerWide.Operations.Configuration
+{
+    public class ServerWideBackupConfiguration : PeriodicBackupConfiguration
+    {
+        internal static string NamePrefix = "Server Wide Backup Configuration";
```

Comment:
Let's add a separator here.

```suggestion
        internal static string NamePrefix = "Server Wide Backup Configuration/";
```

Reply:
I'm adding a comma when adding the task to the database record.
For example:
Server-wide backup name: "Backup to S3".
Backup name in the database: "Server Wide Backup Configuration, Backup to S3"